bot/bot.py

```diff
@@ -44,6 +44,7 @@ def __init__(self, **kwargs):
         self.http_session = ClientSession(
             connector=TCPConnector(resolver=AsyncResolver(), family=socket.AF_INET)
         )
+        self.greeting_task = self.loop.create_task(self.send_log("SeasonalBot", "Connected!"))
```

Comment:
There's no need to store the task; it's never referenced later and it's a fire and forget type of thing.

Reply:
I have a tendency to assign in case something wants to check in on the task later on, but I guess in this case it really isn't necessary. I will remove it.